src/entity/UserAction.ts

```diff
@@ -2,6 +2,11 @@ import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
 
 export enum UserActionType {
   Notification = 'notification',
```

Comment:
This one is wrong in this version right? We described it as `enabled_notification`

Reply:
Yes, this should be fixed in the PR where we utilized this. Was just a placeholder initially.